docs/iceberg-rest-service.md

```diff
@@ -160,40 +145,28 @@ Supports using static GCS credential file or generating GCS token to access GCS
 | Configuration item                                | Description                                                                                        | Default value | Required | Since Version    |
 |---------------------------------------------------|----------------------------------------------------------------------------------------------------|---------------|----------|------------------|
 | `gravitino.iceberg-rest.io-impl`                  | The io implementation for `FileIO` in Iceberg, use `org.apache.iceberg.gcp.gcs.GCSFileIO` for GCS. | (none)        | No       | 0.6.0-incubating |
-| `gravitino.iceberg-rest.credential-provider-type`  | Deprecated, please use `gravitino.iceberg-rest.credential-providers` instead.                      | (none)        | No                                                 | 0.7.0-incubating |
-| `gravitino.iceberg-rest.credential-providers`     | Supports `gcs-token`, generates a temporary token according to the query data path.                | (none)        | No       | 0.7.0-incubating |
-| `gravitino.iceberg-rest.gcs-credential-file-path` | Deprecated, please use `gravitino.iceberg-rest.gcs-service-account-file` instead.                  | (none)        | No       | 0.7.0-incubating |
-| `gravitino.iceberg-rest.gcs-service-account-file` | The location of GCS credential file, only used when `credential-provider-type` is `gcs-token`.     | (none)        | No       | 0.8.0-incubating |
 
 For other Iceberg GCS properties not managed by Gravitino like `gcs.project-id`, you could config it directly by `gravitino.iceberg-rest.gcs.project-id`.
 
-If you set `credential-providers` explicitly, please downloading [Gravitino GCP bundle jar](https://mvnrepository.com/artifact/org.apache.gravitino/gcp-bundle), and place it to the classpath of Iceberg REST server.
+Please refer to [GCS credentials](./security/credential-vending.md#gcs-credentials) for credential related configurations.
 
-Please make sure the credential file is accessible by Gravitino, like using `export GOOGLE_APPLICATION_CREDENTIALS=/xx/application_default_credentials.json` before Gravitino Iceberg REST server is started.
+:::note
+For Gravitino Iceberg REST server, please make sure the credential file is accessible by Gravitino, like using `export GOOGLE_APPLICATION_CREDENTIALS=/xx/application_default_credentials.json` even `gcs-service-account-file` is setting.
```

Comment:
This sentence is not easy to parse.
Please consider revise it.

Reply:
For Gravitino Iceberg REST server, please make sure the credential file is accessible by Gravitino server, you can use environment variants `GOOGLE_APPLICATION_CREDENTIALS`  like `export GOOGLE_APPLICATION_CREDENTIALS=/xx/application_default_credentials.json` to set the value even through configuration `gcs-service-account-file` has already set.

Reply:
updated